views/contact/form.php

```diff
@@ -3,6 +3,7 @@
 /**
  * @var $this \Yiisoft\View\View
  * @var $urlGenerator \Yiisoft\Router\UrlGeneratorInterface
+ * @var $body array
  */
 
 ?>
```

Comment:
In this file `htmlspecialchars` is used. Should be `Html::encode()` instead.

Reply:
Also field values aren't encoded.

Reply:
This file is beautiful, why did I have to touch it? :)